README.md

```diff
@@ -1,27 +1,86 @@
-# Forks and Clones
+# Forking and Cloning Github Repositories
+
+## Problem Statement
+
+If you are familiar with how to create local repositories, you have the ability to
+create a logged history of your project. What's great about open source is that lots
+of people are creating new repositories all the time and sharing their work. You can
+take your repository and push it to a remote to share it, or you can clone a repository
+to your local machine from a remote and make a copy on your own system. Due to permissions
+issues you might need to _fork_ a repository before you _clone_ it. How do you clone from
+a remote to your local system and how to do you fork when necessary?
 
 ## Objectives
 
-1. Fork a repo
-2. Clone a repo
+1. Define _remote_
+2. Use `git clone` to copy a repository to your local machine
+3. Use `git remote` to list remotes
+4. Use `git fork` via GitHub to duplicate other organization's repositories into your own
+
+## Define _remote_
+
+To work with or collaborate on any Git project, you need to be able to manage your _remote_
+repositories. Remote repositories are versions of your project that are hosted online, in this
+case, on Github. With these remote repositories, you'll be pushing and pulling data to and from
+them when you need to share work. To work with remotes, you'll get familiar with _fork_ and
+_clone_, and how they work hand-in-hand with each other, but have two separate meanings
+and functionality.
+
+## Use `git clone` to Copy a Repository to Your Local Machine
+
+Now we want to get our forked version of the repository on our local machine. This
+is where cloning is useful. To do this:
+
+1. Navigate back to the forked repository page you just created on GitHub.
+2. Click the "Clone or Download" green button on the right.
+2. Make sure you select `Use SSH` as your URL type.
+
+	![SSH URL](https://files.readme.io/UgsI2ndmR2aH5ky5G1OA_GitHub%20-%20SSH%20-%201.png)
+
+3. Click the "Copy to clipboard" button (highlighted below). This will copy the
+URL for us to use when we clone.
+
+	![Clone Repo Button](http://readme-pics.s3.amazonaws.com/clone-repo-clone-url-button.png)
+
+4. In the terminal (accessed through the 'Sandbox' or Learn IDE), we need to
+run the `git clone` command. It takes the URL we just copied as an argument,
+like so:
+
+	```bash
+	git clone your-copied-github-url
+	```
+
+This will create a local copy of our forked GitHub repository. 
+
+## Use `git remote` to List Remotes
 
-## Forking a GitHub Repository
+With a repo cloned to your machine, you can now see which remote servers you have configured.
+With `git remote` you'll see the shortnames of each remote handle you’ve specified. If you’ve
```

Comment:
Haven't defined shortname. How would they have specified a remote? They just cloned? They _get_ a remote for free, and that's worth talking about but this material feels like it's not a match to the path we've had them pursue.